crates/uv/src/commands/help.rs

```diff
@@ -30,7 +32,24 @@ pub(crate) fn help(query: &[String], printer: Printer) -> Result<ExitStatus> {
 
     let mut command = command.clone();
     let help = command.render_long_help();
-    writeln!(printer.stderr(), "{}", help.ansi())?;
+
+    let help_ansi = match anstream::Stderr::choice(&std::io::stderr()) {
+        ColorChoice::Always | ColorChoice::AlwaysAnsi => Either::Left(help.ansi()),
+        ColorChoice::Never => Either::Right(help.clone()),
+        // We just asked anstream for a choice, that can't be auto
+        ColorChoice::Auto => unreachable!(),
+    };
+
+    if which("less").is_ok() {
```

Comment:
We should only try this if we're in a tty

Reply:
Yeah that's true, you might not need to do this if the pager checks for you. And it would be very weird for a pager to do anything other than be a `cat` when not connected to a tty.

Reply:
I think we shouldn't do a subprocess call if we don't need to, invocing another program always has it's own caveats

Reply:
I guess.. Cargo does this on every `help` invocation so I'm not particularly concerned but I can change it.